README.md

```diff
@@ -132,37 +132,19 @@ To write events to a file, just redirect your output:
 code42 security-data search -b 2020-02-02 > filename.txt
 ```
 
-To send events to an external server using `netcat` on Linux/Mac:
+To send events to an external server, use the `send-to` command, which behaves exactly the same as `search` but sends
```

Comment:
The one difference between Search is that the default output format is RAW-JSON for send-to and I think it is just JSON for search